### PR TITLE
Update file-manager extension

### PR DESCRIPTION
**Pull request by BalliAsghar**
## Description

- Added Support for setting wallpaper as Desktop Picture

## Screencast

https://github.com/raycast/extensions/assets/14165223/1795c7cc-f288-4655-9a8c-34466e134d3a

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/file-manager/package.json [hunk 1]
```diff
@@ -112,6 +113,7 @@
     "build": "ray build -e dist",
     "dev": "ray develop",
     "lint": "ray lint",
-    "lint:fix": "ray lint --fix"
+    "lint:fix": "ray lint --fix",
```
Comment (erics118) [inline extensions/file-manager/package.json:116]: The name for this script should be `fix-lint`. Do you mind changing it?

**Review by erics118 — COMMENTED** [review]
Thanks for the feature! I never thought to add this, and I think it's pretty useful! Added a few kinda nitpicky comments, but I hope you don't mind 😄

File: extensions/file-manager/src/utils.tsx [hunk 2]
```diff
@@ -158,3 +159,15 @@ export function RenameForm(props: { filePath: string; refresh: () => void; typeN
...
   );
 }
+
+export function isImageFile(file: FileDataType) {
+  const imageExtensions = ["jpg", "jpeg", "png", "gif", "bmp", "tiff", "webp", "heic", "heif"];
+  const extension = file.name.split(".").pop()?.toLowerCase();
+  return extension && imageExtensions.includes(extension);
+}
+
+export const handleSetWallpaper = async (filePath: string) => {
```
[the remark this answers is not in the document]
Comment (BalliAsghar) [reply extensions/file-manager/src/utils.tsx]: Done!

File: extensions/file-manager/src/utils.tsx [hunk 3]
```diff
@@ -158,3 +159,15 @@ export function RenameForm(props: { filePath: string; refresh: () => void; typeN
     </Form>
   );
 }
+
+export function isImageFile(file: FileDataType) {
+  const imageExtensions = ["jpg", "jpeg", "png", "gif", "bmp", "tiff", "webp", "heic", "heif"];
```
[the remark this answers is not in the document]
Comment (BalliAsghar) [reply extensions/file-manager/src/utils.tsx:164]: Most common ones!

File: extensions/file-manager/src/utils.tsx [hunk 4]
```diff
@@ -158,3 +159,15 @@ export function RenameForm(props: { filePath: string; refresh: () => void; typeN
...
+  const extension = file.name.split(".").pop()?.toLowerCase();
+  return extension && imageExtensions.includes(extension);
+}
+
+export const handleSetWallpaper = async (filePath: string) => {
+  await runAppleScript(
+    `tell application "System Events" to tell every desktop to set picture to "${filePath}" as POSIX file`,
```
[the remark this answers is not in the document]
Comment (BalliAsghar) [reply extensions/file-manager/src/utils.tsx]: Fixed!

**Comment by pernielsentikaer** [comment]
Are we good to go @erics118

**Comment by erics118** [comment]
Looks good 👍

**Pull request merged**